sticker-graph/sticker_graph/vendored.py

```diff
@@ -114,10 +115,23 @@ def _create_file_writer_generic_type(logdir,
             flush_millis = tf.constant(2 * 60 * 1000)
         if filename_suffix is None:
             filename_suffix = tf.constant(".v2")
+
+    from pkg_resources import parse_version
+    if parse_version(tf.VERSION) >=  parse_version("1.14"):
+        from tensorflow.python.ops.summary_ops_v2 import ResourceSummaryWriter
+        return ResourceSummaryWriter(
+                shared_name=name,
+                init_op_fn=functools.partial(create_summary_file_writer,
+                logdir=logdir,
+                max_queue=max_queue,
+                flush_millis=flush_millis,
+                filename_suffix=filename_suffix))
+    else:
+        from tensorflow.python.ops.summary_ops_v2 import _make_summary_writer
         return _make_summary_writer(
-            name,
-            create_summary_file_writer,
+             name,
```

Comment:
Are these extra spaces intentional? It's hard to see in GitHub, but it seems that as a result they are not algined with the other arguments anymore? (Again, GitHub may be messing up the layout here.)

Reply:
You're right. Since tensorflow-rocm is not on 1.14 yet, I fixed it remotely using vim and didn't see that.

Should be fine now.